### ISSUE TITLE
Uploading to an Event takes too many clicks

### ISSUE THREAD

**Issue by nums11** [rcos/venue#532]
To upload to an event from the dashboard click the event. 

Underneath the "Your Sections assigned this event" section click view. The upload button appears here.

This second page is unnecessary since it essentially shows the same info as the first one. It could be shortened by having moving upload button and all the other info from the second page to the first page (the page you see when you click the event).

**Comment by jvparin** [comment]
Isn't this the same issue as #518? If it is, it's already covered in this pull request that I made a couple days ago: https://github.com/rcos/venue/pull/529

**Comment by nums11** [comment]
No, this is uploading to an event as a student. #518 is for creating an event as an instructor.
